src/bitcoin-tx.cpp

```diff
@@ -64,6 +64,8 @@ static bool AppInitRawTx(int argc, char* argv[])
         strUsage += HelpMessageOpt("-create", _("Create new, empty TX."));
         strUsage += HelpMessageOpt("-json", _("Select JSON output"));
         strUsage += HelpMessageOpt("-txid", _("Output only the hex-encoded transaction id of the resultant transaction."));
+        strUsage += HelpMessageOpt("-segwit", _("Produce a segwit output."));
```

Comment:
So for instance would you rather transform the `outscript` command such that it accepts the following:
`VALUE:SCRIPT:SEGWIT`
`VALUE:SCRIPT:P2SH`
`VALUE:SCRIPT:SEGWIT:P2SH`
?